### ISSUE TITLE
[Landing Page] #First Section on Home Page

### ISSUE THREAD

**Issue by Nikhil0789** [TaxPlanner/EasyKar.com#5]
Currently landing Page theme images coming on first section. we will update this after sometime

**Comment by Nikhil0789** [comment]
Size of the dark banner should ne reduced

**Comment by Nikhil0789** [comment]
Currently landing Page theme images coming on first section. we will update this after sometime

**Comment by vivekmore** [comment]
not sure what you mean here

**Comment by Nikhil0789** [comment]
Hey Vivek,

It means that as of now we will use the two images for carousel as it is it is there in landing page theme.
If it is required we will change.

**Comment by Nikhil0789** [comment]
First Section
 
![image](https://user-images.githubusercontent.com/11071004/52641231-5d18fb80-2efe-11e9-8a32-ac594c2423c0.png)

**a)	Tag Line on first Carousel Image: Plan Your Holidays
	                                                    While we plan your Tax
b)	Make an Appointment: Replace this with Send your queries**

**Comment by vivekmore** [comment]
there are 2 tag lines in carousel (1st image)
I'll replace the main,

do you want to keep the secondary tag line "Optimize tax ..."?

**Comment by Nikhil0789** [comment]
> there are 2 tag lines in carousel (1st image)
> I'll replace the main,
> 
> do you want to keep the secondary tag line "Optimize tax ..."?

Yes I want to keep Optimize Tax , also While we plan your tax should come same in size as Plan your Holidays
